src/content/docs/victory-group.md

```diff
@@ -67,7 +67,7 @@ categories={["dogs", "cats", "mice"]}
 
 `type: element || array[element]`
 
-`VictoryGroup` works with any combination of the following children: [VictoryArea][], [VictoryBar][], [VictoryCandlestick][], [VictoryErrorBar][], [VictoryLine][], [VictoryScatter][], [VictoryStack][], and [VictoryVoronoi][]. Children supplied to `VictoryGroup` will be cloned and rendered with new props so that all children share common props such as `domain` and `scale`.
+`VictoryGroup` works with any combination of the following children: [VictoryArea][], [VictoryBar][], [VictoryCandlestick][], [VictoryErrorBar][], [VictoryLine][], [VictoryScatter][], [VictoryHistogram][], [VictoryStack][], and [VictoryVoronoi][]. Children supplied to `VictoryGroup` will be cloned and rendered with new props so that all children share common props such as `domain` and `scale`.
```

Comment:
Does `VictoryGroup` work with `VictoryHistogram`?

Reply:
I believe so!

Similar to VictoryChart, just without the axes right?

Something like this seems to work

```
<VictoryGroup
  style={{ data: { fill: "palevioletred" } }}
>
  <VictoryHistogram
    data={sampleHistogramData}
  />
  <VictoryLine data={[{x: 0, y: 0}, {x: 10, y: 10}]} />
</VictoryGroup>
```

Reply:
Ah gotcha, yeah! It won't work with group offsets like `VictoryBar` though.